[2024-11-14T20:25:37.998+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:25:38.005+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:25:38.088+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:25:38.088+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:25:38.353+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:25:38.338+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:25:38.355+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:25:38.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.487 seconds
[2024-11-14T20:26:08.871+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:08.893+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:26:08.899+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:26:08.899+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:08.930+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:26:08.925+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:26:08.932+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:08.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.119 seconds
[2024-11-14T20:26:39.311+0000] {processor.py:186} INFO - Started process (PID=83) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:39.313+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:26:39.317+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:26:39.316+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:39.360+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:26:39.354+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:26:39.361+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:26:39.392+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.089 seconds
[2024-11-14T20:27:09.586+0000] {processor.py:186} INFO - Started process (PID=90) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:09.587+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:27:09.591+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:27:09.590+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:09.604+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:27:09.600+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:27:09.606+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:09.626+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.047 seconds
[2024-11-14T20:27:29.835+0000] {processor.py:186} INFO - Started process (PID=92) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:29.836+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:27:29.839+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:27:29.839+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:29.863+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:27:29.860+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:27:29.865+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:27:29.882+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.052 seconds
[2024-11-14T20:28:00.151+0000] {processor.py:186} INFO - Started process (PID=101) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:00.152+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:28:00.155+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:00.155+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:00.174+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:00.169+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 5, in <module>
    from airlfow.operators.python import PythonOperator
ModuleNotFoundError: No module named 'airlfow'
[2024-11-14T20:28:00.176+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:00.199+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.054 seconds
[2024-11-14T20:28:07.260+0000] {processor.py:186} INFO - Started process (PID=108) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:07.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:28:07.264+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:07.264+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:07.283+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:07.279+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:28:07.284+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:07.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.047 seconds
[2024-11-14T20:28:37.657+0000] {processor.py:186} INFO - Started process (PID=117) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:37.658+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:28:37.660+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:37.660+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:37.674+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:28:37.671+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:28:37.675+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:28:37.704+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.052 seconds
[2024-11-14T20:29:08.389+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:08.390+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:29:08.393+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:29:08.393+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:08.406+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:29:08.402+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:29:08.407+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:08.426+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.042 seconds
[2024-11-14T20:29:38.688+0000] {processor.py:186} INFO - Started process (PID=133) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:38.689+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:29:38.691+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:29:38.691+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:38.704+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:29:38.700+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:29:38.705+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:29:38.732+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.050 seconds
[2024-11-14T20:30:05.095+0000] {processor.py:186} INFO - Started process (PID=136) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:05.096+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:30:05.100+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:05.099+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:05.123+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:05.118+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL-DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL-DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:30:05.124+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:05.141+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.053 seconds
[2024-11-14T20:30:06.155+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:06.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:30:06.161+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:06.160+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:06.201+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:06.193+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL-DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL-DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:30:06.203+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:06.240+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.102 seconds
[2024-11-14T20:30:16.365+0000] {processor.py:186} INFO - Started process (PID=144) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:16.367+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:30:16.377+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:16.376+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:16.405+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:16.401+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL-DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL-DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:30:16.406+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:16.428+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.073 seconds
[2024-11-14T20:30:17.398+0000] {processor.py:186} INFO - Started process (PID=145) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:17.405+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:30:17.416+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:17.415+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:17.457+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:17.449+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL-DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL-DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:30:17.459+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:17.514+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.125 seconds
[2024-11-14T20:30:47.914+0000] {processor.py:186} INFO - Started process (PID=153) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:47.916+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:30:47.918+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:47.918+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:47.935+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:30:47.931+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL-DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL-DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:30:47.936+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:30:47.953+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.047 seconds
[2024-11-14T20:31:14.516+0000] {processor.py:186} INFO - Started process (PID=163) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:14.517+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:31:14.523+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:31:14.523+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:14.550+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:31:14.543+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_ETL.py", line 38, in <module>
    with DAG(dag_id='ETL_DATA ',
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 593, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/helpers.py", line 57, in validate_key
    raise AirflowException(
airflow.exceptions.AirflowException: The key 'ETL_DATA ' has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2024-11-14T20:31:14.551+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:14.573+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.063 seconds
[2024-11-14T20:31:35.745+0000] {processor.py:186} INFO - Started process (PID=164) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:35.747+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:31:35.751+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:31:35.751+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:35.779+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:31:35.776+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:31:35.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:31:35.818+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.087 seconds
[2024-11-14T20:32:06.132+0000] {processor.py:186} INFO - Started process (PID=172) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:06.133+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:32:06.135+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:32:06.135+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:06.146+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:32:06.144+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:32:06.147+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:06.188+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.061 seconds
[2024-11-14T20:32:36.454+0000] {processor.py:186} INFO - Started process (PID=180) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:36.455+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:32:36.458+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:32:36.457+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:36.474+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:32:36.471+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:32:36.475+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:32:36.506+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.058 seconds
[2024-11-14T20:33:06.826+0000] {processor.py:186} INFO - Started process (PID=188) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:06.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:33:06.830+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:33:06.830+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:06.847+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:33:06.845+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:33:06.847+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:06.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.058 seconds
[2024-11-14T20:33:30.131+0000] {processor.py:186} INFO - Started process (PID=196) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:30.132+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:33:30.136+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:33:30.135+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:30.162+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:33:30.159+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:33:30.163+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:33:30.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.062 seconds
[2024-11-14T20:34:00.529+0000] {processor.py:186} INFO - Started process (PID=205) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:00.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:34:00.533+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:00.533+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:00.554+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:00.552+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:34:00.555+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:00.575+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.053 seconds
[2024-11-14T20:34:30.860+0000] {processor.py:186} INFO - Started process (PID=213) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:30.861+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:34:30.863+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:30.863+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:30.878+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:30.876+0000] {dagbag.py:484} ERROR - Failed to bag_dag: /opt/airflow/dags/dag_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 102, in validate
    croniter(self._expression)
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 217, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(
                                               ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 943, in expand
    return cls._expand(expr_format, hash_id=hash_id,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/croniter/croniter.py", line 760, in _expand
    raise CroniterBadCronError(cls.bad_length)
croniter.croniter.CroniterBadCronError: Exactly 5, 6 or 7 columns has to be specified for iterator expression.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 479, in _process_modules
    dag.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 845, in validate
    self.timetable.validate()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/timetables/_cron.py", line 104, in validate
    raise AirflowTimetableInvalid(str(e))
airflow.exceptions.AirflowTimetableInvalid: Exactly 5, 6 or 7 columns has to be specified for iterator expression.
[2024-11-14T20:34:30.878+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:30.894+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.040 seconds
[2024-11-14T20:34:54.146+0000] {processor.py:186} INFO - Started process (PID=221) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:54.147+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:34:54.149+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.149+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:54.168+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:54.267+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.267+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:ETL_DATA
[2024-11-14T20:34:54.276+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.276+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:ETL_DATA
[2024-11-14T20:34:54.282+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.282+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:ETL_DATA
[2024-11-14T20:34:54.289+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.289+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:ETL_DATA
[2024-11-14T20:34:54.294+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.294+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:ETL_DATA
[2024-11-14T20:34:54.301+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.300+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:ETL_DATA
[2024-11-14T20:34:54.308+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.308+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:ETL_DATA
[2024-11-14T20:34:54.309+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.309+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:34:54.319+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.319+0000] {dag.py:3262} INFO - Creating ORM DAG for ETL_DATA
[2024-11-14T20:34:54.320+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:54.320+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:34:54.342+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.201 seconds
[2024-11-14T20:34:56.397+0000] {processor.py:186} INFO - Started process (PID=222) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:56.398+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:34:56.400+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:56.400+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:56.420+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:56.429+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:56.429+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:34:56.442+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:56.441+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:34:56.464+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.072 seconds
[2024-11-14T20:34:57.422+0000] {processor.py:186} INFO - Started process (PID=223) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:57.423+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:34:57.425+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:57.425+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:57.443+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:34:57.455+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:57.454+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:34:57.471+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:34:57.470+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:34:57.507+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.092 seconds
[2024-11-14T20:35:27.740+0000] {processor.py:186} INFO - Started process (PID=231) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:27.741+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:35:27.743+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:27.743+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:27.757+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:27.791+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:27.791+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:35:27.804+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:27.804+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:35:27.835+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.101 seconds
[2024-11-14T20:35:58.168+0000] {processor.py:186} INFO - Started process (PID=239) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:58.171+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:35:58.174+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:58.174+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:58.192+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:35:58.233+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:58.233+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:35:58.255+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:35:58.254+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:35:58.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.142 seconds
[2024-11-14T20:36:28.559+0000] {processor.py:186} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:28.560+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:36:28.563+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:28.563+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:28.576+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:28.609+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:28.608+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:36:28.632+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:28.632+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:36:28.670+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.116 seconds
[2024-11-14T20:36:58.941+0000] {processor.py:186} INFO - Started process (PID=255) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:58.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:36:58.944+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:58.944+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:58.957+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:36:58.980+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:58.980+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:36:58.995+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:36:58.995+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:36:59.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.090 seconds
[2024-11-14T20:37:29.337+0000] {processor.py:186} INFO - Started process (PID=263) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:29.337+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:37:29.340+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:29.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:29.355+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:29.389+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:29.389+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:37:29.416+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:29.415+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:37:29.468+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.137 seconds
[2024-11-14T20:37:59.754+0000] {processor.py:186} INFO - Started process (PID=273) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:59.755+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:37:59.759+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:59.759+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:59.787+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:37:59.835+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:59.834+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:37:59.866+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:37:59.865+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:37:59.906+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.161 seconds
[2024-11-14T20:38:30.398+0000] {processor.py:186} INFO - Started process (PID=282) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:38:30.399+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:38:30.402+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:38:30.402+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:38:30.417+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:38:30.444+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:38:30.444+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:38:30.458+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:38:30.458+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:38:30.480+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.088 seconds
[2024-11-14T20:39:00.941+0000] {processor.py:186} INFO - Started process (PID=290) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:00.944+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:39:00.956+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:00.955+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:01.005+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:01.049+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:01.049+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:39:01.070+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:01.069+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:39:01.093+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.161 seconds
[2024-11-14T20:39:31.463+0000] {processor.py:186} INFO - Started process (PID=298) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:31.464+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:39:31.468+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:31.467+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:31.487+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:39:31.529+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:31.528+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:39:31.545+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:39:31.545+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:39:31.565+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.112 seconds
[2024-11-14T20:40:01.849+0000] {processor.py:186} INFO - Started process (PID=306) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:01.851+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:40:01.853+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:01.853+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:01.870+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:01.893+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:01.893+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:40:01.908+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:01.908+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:40:01.939+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.096 seconds
[2024-11-14T20:40:32.241+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:32.242+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:40:32.245+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:32.245+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:32.259+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:40:32.285+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:32.284+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:40:32.298+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:40:32.297+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:40:32.342+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.106 seconds
[2024-11-14T20:41:02.716+0000] {processor.py:186} INFO - Started process (PID=322) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:02.717+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:41:02.720+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:02.719+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:02.739+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:02.774+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:02.774+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:41:02.795+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:02.794+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:41:02.820+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.111 seconds
[2024-11-14T20:41:33.642+0000] {processor.py:186} INFO - Started process (PID=330) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:33.644+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:41:33.659+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:33.659+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:33.703+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:41:33.773+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:33.773+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:41:33.818+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:41:33.817+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:41:33.898+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.267 seconds
[2024-11-14T20:47:14.686+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:14.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:47:14.692+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:14.691+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:14.720+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:15.158+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.157+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:ETL_DATA
[2024-11-14T20:47:15.179+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.178+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:ETL_DATA
[2024-11-14T20:47:15.200+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.199+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:ETL_DATA
[2024-11-14T20:47:15.228+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.225+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:ETL_DATA
[2024-11-14T20:47:15.243+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.243+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:ETL_DATA
[2024-11-14T20:47:15.257+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.257+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:ETL_DATA
[2024-11-14T20:47:15.268+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.268+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:ETL_DATA
[2024-11-14T20:47:15.269+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.269+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:47:15.291+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.290+0000] {dag.py:3262} INFO - Creating ORM DAG for ETL_DATA
[2024-11-14T20:47:15.292+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:15.291+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:47:15.324+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.644 seconds
[2024-11-14T20:47:45.688+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:45.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:47:45.694+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:45.693+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:45.719+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:47:45.754+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:45.753+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:47:45.777+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:47:45.776+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:47:45.805+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.125 seconds
[2024-11-14T20:48:16.153+0000] {processor.py:186} INFO - Started process (PID=88) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:16.154+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:48:16.157+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:16.157+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:16.170+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:16.195+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:16.195+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:48:16.212+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:16.211+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:48:16.237+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.089 seconds
[2024-11-14T20:48:46.414+0000] {processor.py:186} INFO - Started process (PID=96) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:46.418+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:48:46.424+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:46.424+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:46.449+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:48:46.487+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:46.487+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:48:46.510+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:48:46.510+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:48:46.554+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.150 seconds
[2024-11-14T20:49:16.901+0000] {processor.py:186} INFO - Started process (PID=103) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:16.903+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:49:16.906+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:16.905+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:16.930+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:17.007+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:17.007+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:49:17.054+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:17.053+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:49:17.116+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.224 seconds
[2024-11-14T20:49:47.357+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:47.358+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:49:47.362+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:47.361+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:47.375+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:49:47.397+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:47.397+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:49:47.410+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:49:47.410+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:49:47.429+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.077 seconds
[2024-11-14T20:50:17.734+0000] {processor.py:186} INFO - Started process (PID=119) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:17.735+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:50:17.738+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:17.737+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:17.752+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:17.776+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:17.776+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:50:17.791+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:17.791+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:50:17.819+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.092 seconds
[2024-11-14T20:50:48.010+0000] {processor.py:186} INFO - Started process (PID=127) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:48.011+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:50:48.013+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:48.013+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:48.026+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:50:48.050+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:48.050+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:50:48.062+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:50:48.062+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:50:48.093+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.089 seconds
[2024-11-14T20:51:18.294+0000] {processor.py:186} INFO - Started process (PID=135) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:18.295+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:51:18.297+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:18.296+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:18.310+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:18.332+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:18.332+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:51:18.344+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:18.344+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:51:18.375+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.086 seconds
[2024-11-14T20:51:48.571+0000] {processor.py:186} INFO - Started process (PID=143) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:48.572+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:51:48.576+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:48.575+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:48.596+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:51:48.631+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:48.630+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:51:48.651+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:51:48.650+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:51:48.670+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.104 seconds
[2024-11-14T20:52:18.976+0000] {processor.py:186} INFO - Started process (PID=151) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:18.977+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:52:18.979+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:18.979+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:18.994+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:19.039+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:19.038+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:52:19.066+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:19.066+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:52:19.097+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.126 seconds
[2024-11-14T20:52:49.525+0000] {processor.py:186} INFO - Started process (PID=159) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:49.526+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:52:49.528+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:49.528+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:49.549+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:52:49.574+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:49.573+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:52:49.586+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:52:49.586+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:52:49.604+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.086 seconds
[2024-11-14T20:53:19.791+0000] {processor.py:186} INFO - Started process (PID=167) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:19.792+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:53:19.795+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:19.794+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:19.809+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:19.835+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:19.835+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:53:19.849+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:19.849+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:53:19.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.087 seconds
[2024-11-14T20:53:50.124+0000] {processor.py:186} INFO - Started process (PID=175) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:50.125+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:53:50.128+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:50.127+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:50.145+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:53:50.176+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:50.175+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:53:50.194+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:53:50.194+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:53:50.219+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.100 seconds
[2024-11-14T20:54:20.450+0000] {processor.py:186} INFO - Started process (PID=184) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:20.451+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:54:20.455+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:20.455+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:20.479+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:20.530+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:20.530+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:54:20.563+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:20.562+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:54:20.608+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.169 seconds
[2024-11-14T20:54:50.923+0000] {processor.py:186} INFO - Started process (PID=197) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:50.925+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:54:50.934+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:50.928+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:50.967+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:54:51.016+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:51.016+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:54:51.031+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:54:51.030+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:54:51.056+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.142 seconds
[2024-11-14T20:55:21.337+0000] {processor.py:186} INFO - Started process (PID=205) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:21.338+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:55:21.341+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:21.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:21.356+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:21.384+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:21.384+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:55:21.398+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:21.398+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:55:21.421+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.090 seconds
[2024-11-14T20:55:51.895+0000] {processor.py:186} INFO - Started process (PID=213) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:51.903+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:55:51.907+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:51.907+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:51.943+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:55:51.982+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:51.982+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:55:52.002+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:55:52.002+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:55:52.040+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.151 seconds
[2024-11-14T20:56:22.356+0000] {processor.py:186} INFO - Started process (PID=221) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:22.357+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:56:22.360+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:22.359+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:22.377+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:22.407+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:22.407+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:56:22.424+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:22.423+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:56:22.458+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.108 seconds
[2024-11-14T20:56:52.757+0000] {processor.py:186} INFO - Started process (PID=230) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:52.758+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:56:52.762+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:52.761+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:52.779+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:56:52.809+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:52.808+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:56:52.829+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:56:52.829+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:56:52.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.129 seconds
[2024-11-14T20:57:23.225+0000] {processor.py:186} INFO - Started process (PID=238) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:23.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:57:23.229+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:23.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:23.244+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:23.268+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:23.267+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:57:23.284+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:23.284+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:57:23.308+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.089 seconds
[2024-11-14T20:57:53.592+0000] {processor.py:186} INFO - Started process (PID=246) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:53.594+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:57:53.600+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:53.599+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:53.625+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:57:53.659+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:53.658+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:57:53.679+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:57:53.679+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:57:53.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.139 seconds
[2024-11-14T20:58:23.899+0000] {processor.py:186} INFO - Started process (PID=254) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:23.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:58:23.902+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:23.902+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:23.916+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:23.942+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:23.942+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:58:23.958+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:23.958+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:58:23.988+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.095 seconds
[2024-11-14T20:58:54.229+0000] {processor.py:186} INFO - Started process (PID=262) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:54.231+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:58:54.235+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:54.234+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:54.251+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:58:54.282+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:54.281+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:58:54.298+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:58:54.298+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:58:54.320+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.099 seconds
[2024-11-14T20:59:24.633+0000] {processor.py:186} INFO - Started process (PID=275) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:24.634+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:59:24.637+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:24.636+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:24.650+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:24.677+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:24.677+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:59:24.691+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:24.691+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:59:24.716+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.088 seconds
[2024-11-14T20:59:54.995+0000] {processor.py:186} INFO - Started process (PID=283) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:54.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T20:59:55.000+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:54.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:55.026+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T20:59:55.084+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:55.083+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T20:59:55.111+0000] {logging_mixin.py:190} INFO - [2024-11-14T20:59:55.111+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T20:59:55.142+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.155 seconds
[2024-11-14T21:00:25.457+0000] {processor.py:186} INFO - Started process (PID=290) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:25.458+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T21:00:25.473+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:25.472+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:25.515+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:25.564+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:25.563+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T21:00:25.592+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:25.592+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T21:00:25.628+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.189 seconds
[2024-11-14T21:00:56.006+0000] {processor.py:186} INFO - Started process (PID=298) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:56.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T21:00:56.011+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:56.011+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:56.030+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:00:56.063+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:56.063+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T21:00:56.086+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:00:56.086+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T21:00:56.133+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.135 seconds
[2024-11-14T21:01:26.275+0000] {processor.py:186} INFO - Started process (PID=306) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:26.277+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T21:01:26.281+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:26.280+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:26.310+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:26.339+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:26.339+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T21:01:26.356+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:26.356+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T21:01:26.389+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.121 seconds
[2024-11-14T21:01:56.623+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:56.624+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_ETL.py for tasks to queue
[2024-11-14T21:01:56.629+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:56.629+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:56.643+0000] {processor.py:925} INFO - DAG(s) 'ETL_DATA' retrieved from /opt/airflow/dags/dag_ETL.py
[2024-11-14T21:01:56.672+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:56.672+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-14T21:01:56.686+0000] {logging_mixin.py:190} INFO - [2024-11-14T21:01:56.685+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_DATA to None, run_after=None
[2024-11-14T21:01:56.715+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_ETL.py took 0.100 seconds
